### PR TITLE
Added security auditor role to docs

### PR DESCRIPTION
**Pull request by thehansys**
Description for a new organization role that is available on edgio.app

### CODE REVIEW

File: guides/applications/v7/basics/collaboration.md [hunk 1]
```diff
@@ -55,9 +55,10 @@ Organization management consists of:
...
+|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Viewer           | This role grants read-only access. <br /><br />If applied at the organization level, then the user will have read-only access to the organization's settings, properties, and environments.                                                             |
 | Purger           | This role grants the Viewer role along with the ability to purge content. <br /><br />If applied at the organization level, then the user may purge content for any of the organization's properties.                                                   |
+| Security Auditor | This role grants the Viewer role along with the ability to view and comment all security-related features.                                                                                                                                              |
```
Comment (jsernaque) [inline guides/applications/v7/basics/collaboration.md:61]:
```suggestion
| Security Auditor | This role grants the Viewer role along with the ability to view all security-related features.                                                                                                                                              |
```

Comment (thehansys) [reply guides/applications/v7/basics/collaboration.md:61]: @jsernaque Comments are here. I'll keep it up to you if you think it's worth mentioning it in the role description.
MicrosoftTeams-image (4)

Comment (jsernaque) [reply guides/applications/v7/basics/collaboration.md:61]: @thehansys If commenting is currently restricted to an unreleased feature (i.e., ASM), we should remove it from the description.

Comment (thehansys) [reply guides/applications/v7/basics/collaboration.md]: @jsernaque Updated 👍

**Review by jsernaque — APPROVED** [review]

**Pull request merged**